en/basic-concepts/environments-to-organize-workloads-and-users.md

```diff
@@ -27,10 +27,10 @@ Any user with the **User** role, or any other role which has the *Environments:
 1. If you wish to allow users from other organizations to be added as members, select **Allow external members**.
 1. Click *Submit*.  The environment will be provisioned.
```

Comment:
```suggestion
1. Click *Next*.  The environment will be provisioned.
```

Reply:
Holy crap, did that change?  Maybe I am confused, I remember it being Submit.  But you're right, it's Next.  Making the change.

Reply:
that change was with the long running operations